snapcraft/cli/_errors.py

```diff
@@ -46,6 +59,9 @@ def exception_handler(exception_type, exception, exception_traceback, *,
         traceback.print_exception(
             exception_type, exception, exception_traceback)
 
+    if not is_snapcraft_error and RavenClient is not None:
+        _submit_trace(exception)
```

Comment:
Can we have a test case for this? It seems like there's three cases here at least that we'll care about.

1. Trace prompt will be triggered
2. Non-interactive build skips past the prompt (rather than timing out)
3. No trace available